Yafc.UI/ImGui/ScrollArea.cs

```diff
@@ -74,20 +74,23 @@ public abstract class Scrollable(bool vertical, bool horizontal, bool collapsibl
             }
             else {
                 if (horizontal && maxScroll.X > 0f) {
-                    Rect fullScrollRect = new Rect(rect.X, rect.Bottom - ScrollbarSize, rect.Width, ScrollbarSize);
+                    Rect fullScrollRect = new Rect(rect.X, rect.Bottom, rect.Width, ScrollbarSize);
                     Rect scrollRect = new Rect(rect.X + scrollStart.X, fullScrollRect.Y, scrollSize.X, ScrollbarSize);
-                    BuildScrollBar(gui, 0, in fullScrollRect, in scrollRect);
+                    // The horizontal scrollbar is drawn '1 scrollbar' too high, so fix to match mouse click tests
```

Comment:
When I tested, I found that the hit testing was off by `ScrollbarSize - .5f`, not `ScrollbarSize`. This was more obvious when I increased ScrollbarSize to 4; with that, only the top sliver of scrollbar worked on master, and all but the top sliver works with this PR.

The hit-test for the vertical scrollbar is also off by 0.5 vertical units. It seems like those should be related, but I don't know if they are.

Reply:
If vertical is also off by a bit, then the 'exception I had to make for the horizontal scrollbar is not an exception but something that goes for both (which I like better :wink: ).
Good idea to increase the scrollbar size for easier/better testing. I'll do that as well and try to find a better cause/fix!

Reply:
Also, the Summary page needs to guess the area (mostly height) of the ScrollArea it creates, as I cannot seem to find a suitable way of determining the available height of the remaining/empty space.
When I increase the scrollbar size, it becomes apparent that it is slightly to large (the horizontal scrollbar is placed a little too low). Although this should not influence the 'clickable area'/'click test'

Reply:
I fully redid the PR, so I'll mark this comment/thread as resolved as it is completely outdated.